### PR TITLE
Add card--overlay

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add the card--overlay pattern.

## QA
- Run `gulp jekyll`
- Go to http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/card/overlay/
- Make sure it matches [the design spec](https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Card/card-overlay.md).

## Details
Fixes https://github.com/ubuntudesign/vanilla-brochure-theme/issues/2

### CODE REVIEW

File: scss/_patterns_card.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@mixin theme-p-card {
+  @include theme-p-card--overlay;
+}
+
+// Pattern should be created for strips with bg images
+@mixin theme-p-card--overlay {
+  .p-card--overlay {
+    $bg-color: rgba($color-x-light, .85);
+    background: $bg-color;
+    padding: 1.333rem;
```
Comment (barrymcgee) [inline scss/_patterns_card.scss]: Make this `padding: 1.3333rem;` to make it exactly `20px` when computed.

File: scss/_patterns_card.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Brochure themes card styling
+@mixin theme-p-card {
+  @include theme-p-card--overlay;
+}
+
+// Pattern should be created for strips with bg images
+@mixin theme-p-card--overlay {
+  .p-card--overlay {
```
Comment (barrymcgee) [inline scss/_patterns_card.scss]: Add carriage return for consistency.

**Comment by barrymcgee** [comment]
screen shot 2017-02-15 at 16 20 46

**Comment by anthonydillon** [comment]
Thanks @barrymcgee, can you pull again

**Pull request merged**